Helper/Refund.php

```diff
@@ -29,8 +29,9 @@ public function createRefundRequest($order, $amount) {
             "shopConfig" => $this->config->getShopKey(),
             "refundAmount" => $amount,
             "refundCurrency" => $order->getOrderCurrencyCode(),
-            "description" => "Refund for order #".$order->getExtOrderId(),
+            "description" => "Refund for order #".$order->getData('purchaseId'),
```

Comment:
please double check the order needed, either `merchantReferenceId` or `getRealOrderId()` but maybe is not the purchaseId right?

Reply:
this field is shown to the user when paying, we can keep current implementation as getRealOrderId will return a db id which won't be an interest for the user